src/styles/Button.module.scss

```diff
@@ -1,42 +1,49 @@
 @import './Constants.scss';
 
+@mixin disableDefaultButtonStyles {
+  font-family: inherit;
+  background: none;
+  border: none;
+  padding: 0;
+}
+
+$hover-transition-time: 0.2s;
+$click-transition-time: 0.01s;
+$button-height: 3.25rem;
+
 .background {
   background-color: black;
 }
 
 .button {
-  background: none;
-	border: none;
-	padding: 0;
-	outline: inherit;
-  display: inline-block;
-  margin: 1rem 1rem;
-  box-sizing: border-box;
+  @include disableDefaultButtonStyles;
   cursor: pointer;
-  width: fit-content;
+  background: none;
+  margin: 1rem;
   font-weight: 700;
```

Comment:
ten width: fit-content to naprawdę nie wiem na co, jego usunięcie nic nie zmieniło